src/lib/rest/rest.cpp

```diff
@@ -1239,6 +1245,23 @@ static int connectionTreat
       return MHD_NO;
     }
 
+
+    // Get API version
+    ciP->apiVersion = (url[2] == '2')? V2 : V1;  // If an APIv2 request, the URL starts with "/v2/". Only V2 requests.
+
+    // LM_TMP(("--------------------- Serving APIv%d request %s %s -----------------", ciP->apiVersion, method, url));
+
+    // Lookup Rest Service
+    bool badVerb = false;
+    ciP->restServiceP = restServiceLookup(ciP, &badVerb);
+
+    if (badVerb)
+    {
+      // Bad Verb is taken care of later
+      ciP->httpStatusCode = SccBadVerb;
+      ciP->restServiceP   = &restServiceForBadVerb;  // FIXME PR: Try to remove this, or make restServiceLookup return a dummy
```

Comment:
FIXME PR to be solved in this same PR or in a next one in the #3109 series?

Reply:
My intention (obviously) was to fix it inside the PR.
However, I have a new PR on its way with this "problem" already taken care of.
Changing 'FIXME PR' for 'FIXME P4'

Reply:
FIXME #3109-PR could be also a good alternative

Reply:
ok ...
Not sure though what you mean with `#3109-PR` ...

Reply:
Fixed in 1e2e62c